app/components/CardBet.tsx

```diff
@@ -1,39 +1,71 @@
 "use client";
 
 import { Card } from "@/app/components/ui";
-import { FixedProductMarketMaker } from "@/queries/omen";
+import { UserPosition } from "@/queries/conditional-tokens/types";
+import { FixedProductMarketMaker, getConditionMarket } from "@/queries/omen";
 import { remainingTime } from "@/utils/dates";
+import { useQuery } from "@tanstack/react-query";
 import { cx } from "class-variance-authority";
 import Link from "next/link";
 import { Button, Logo, Tag } from "swapr-ui";
 
-interface CardBetProps {
-  market: FixedProductMarketMaker;
+interface BetProps {
+  userPosition: UserPosition;
 }
 
-export const CardBet = ({ market }: CardBetProps) => {
+export const CardBet = ({ userPosition }: BetProps) => {
+  const conditionId = userPosition.position.conditionIdsStr;
+
+  const { data, isLoading, isFetching } = useQuery({
+    queryKey: ["getConditionMarket", conditionId],
+    queryFn: async () =>
+      getConditionMarket({
+        id: conditionId,
+      }),
+    enabled: !!conditionId,
+  });
+
+  if (isLoading || isFetching) return <LoadingCardBet />;
+
+  const market: FixedProductMarketMaker = data?.conditions[0]
+    ?.fixedProductMarketMakers[0] as FixedProductMarketMaker;
+
   const closingDate = new Date(+market.openingTimestamp * 1000);
+  const answer = market.question?.currentAnswer;
+
+  const outcomeIndex = userPosition.position.indexSets[0];
 
-  const outcomes = ["win", "loss", "-"];
-  const randomOutcome = outcomes[Math.floor(Math.random() * outcomes.length)];
+  const outcomes = userPosition.position.conditions[0].outcomes;
+  const outcomeString = outcomes ? outcomes[outcomeIndex - 1] : "";
+
+  const winner = answer === "win";
+  const loser = answer === "loss";
+
+  const outcomeAmountString = answer
+    ? winner
+      ? "You won"
+      : "You lost"
+    : "Potential win";
 
   return (
     <Card
       className={cx(
-        "w-full bg-gradient-to-b from-[#131313]",
-        randomOutcome === "win" && "from-[#131313] to-[#11301F]",
-        randomOutcome === "loss" && "from-[#131313] to-[#301111]"
+        "w-full bg-gradient-to-b from-[#F1F1F1] dark:from-[#131313]",
+        winner &&
+          "from-[#F2f2F2] to-[#d0ffd6] dark:from-[#131313] dark:to-[#11301F]",
+        loser &&
+          "from-[#F2f2F2] to-[#f4cbc4] dark:from-[#131313] dark:to-[#301111]"
```

Comment:
Same here here. If it `hasFinished` and `isWinner` show one gradient, otherwise show the other.